modules/management-console-agent/src/main/java/org/apache/ignite/agent/dto/cache/CacheInfo.java

```diff
@@ -29,6 +29,9 @@
     /** Deployment id. */
     private IgniteUuid deploymentId;
 
+    /** Created by sql. */
+    private boolean createdBySql;
```

Comment:
Why this flag has different name in compare to ignite-core?

Reply:
The core name will only confuses us, because initially I thought that sql flag mean availability of sql queries for this cache.